src/components/Snackbar/Snackbar.js

```diff
@@ -14,6 +14,11 @@ class Snackbar extends Component {
      */
     status: PropTypes.oneOf(['default', 'success', 'danger']),
 
+    /**
+     * Snackbar max-width in rem.
```

Comment:
Technically with a number passed through, this becomes a pixel value, which is fine